src/Components/TooltipCallout/TooltipCallout.tsx

```diff
@@ -38,35 +37,33 @@ const TooltipCallout: React.FC<ITooltipCalloutProps> = (props) => {
     });
 
     return (
-        <div className={classNames.root}>
-            <IconButton
-                ariaLabel={buttonAriaLabel}
-                data-testid={dataTestId}
-                id={id}
-                iconProps={{ iconName: iconName || 'Info' }}
-                onFocus={toggleFlyout}
-                onBlur={toggleFlyout}
-                onMouseEnter={toggleFlyout}
-                onMouseLeave={toggleFlyout}
-                styles={classNames.subComponentStyles.button()}
-            />
-            {flyoutVisible && (
-                <Callout
-                    directionalHint={DirectionalHint.rightCenter}
-                    {...calloutProps}
-                    target={`#${id}`}
-                    onDismiss={toggleFlyout}
-                    styles={classNames.subComponentStyles.callout}
-                >
-                    {calloutContent}
-                    {link && ' '}
-                    {link && (
-                        <Link target="_blank" href={link.url}>
-                            {link.text || t('learnMore')}
-                        </Link>
-                    )}
-                </Callout>
-            )}
+        <div>
+            <TooltipHost
+                directionalHint={DirectionalHint.rightCenter}
+                {...calloutProps}
+                delay={TooltipDelay.zero}
+                styles={classNames.subComponentStyles.callout}
+                content={
+                    <div style={{ fontSize: '14px' }}>
```

Comment:
do we need the wrapping div? could we set the style on the tooltip styles object instead?

Reply:
div here is needed b/c content prop is of type string | JSX Element -- I also tried moving the font size to the tooltip styles object but no dice